### PR TITLE
Using style helpers guide

### PR DESCRIPTION
**Pull request by borja-munoz**
Initial guide with examples

### CODE REVIEW

File: docs/guides/02-using-style-helpers.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+
+* color
+* size (only for points and lines)
+* opacity
+* strokeColor (only for points and polygons)
+* strokeWidth (only for points and polygons)
+
+Each of these options have different defaults depending on the type of geometry. For instance, the default color for points is #EE4D5A, #4CC8A3 for lines and #826DBA for polygons. For a complete list of all the default values for the different properties, please go to the [reference](ToDo Add link).
+
+If we use the basic style helper without specifying options, we will obtain the same result that if we were not using the helper.
```
Comment (VictorVelarde) [inline docs/guides/02-using-style-helpers.md]: picky thing: if we use from the beginning "you will learn", "you xxx", I would keep the voice here using "If you use..." / "You will obtain"

File: docs/guides/02-using-style-helpers.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
+## Using style helpers
+
+In this guide, you will learn how to create insightful visualizations with few lines of code using the Web SDK style helpers. It is recommended to have read the Getting Started guide to understand the basic structure of a Web SDK application.
```
Comment (VictorVelarde) [inline docs/guides/02-using-style-helpers.md]: How about adding somewhere a note like this?

"Web SDK uses internally deck.gl, so you can always use pure deck.gl styles if you prefer; see for example this layer style here (TODO: links to deck.gl style example / deck.gl GeoJSON reference). But with the style helpers you can get more easily some common layer visualizations (like for example creating a bubble map taking into account different classification methods for your bins)"

File: docs/guides/02-using-style-helpers.md [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+</div>
+> View this step [here](/developers/web-sdk/examples/maps/guides/using-style-helpers/step-5.html)
+
+### Size style helpers
+
+Similar to the color style helpers, you can use the size style helpers to easily change the size ot the symbology representing the features depending on the values of a given feature property. If the property data type is numeric, you can group the features in bins using the `carto.viz.style.sizeBins` helper. If the property data type is a string representing values from a data domain, you can group the features in categories using the `carto.viz.style.sizeCategories` helper. There is an additional helper `carto.viz.style.sizeContinuous` that you can use for creating a continuous color ramp for numeric feature properties.
```
Comment (VictorVelarde) [inline docs/guides/02-using-style-helpers.md]: typo in "continuous color ramp" --> "continuous size" + "interpolation" ?

**Review by VictorVelarde — APPROVED** [review]
I really liked the approach ✨ . I think this is a great guide. Just some small comments 🚀

File: docs/guides/02-using-style-helpers.md (hunk 1 above)
Comment (borja-munoz) [reply docs/guides/02-using-style-helpers.md]: Fixed

File: docs/guides/02-using-style-helpers.md (hunk 3 above)
Comment (borja-munoz) [reply docs/guides/02-using-style-helpers.md]: Fixed

File: docs/guides/02-using-style-helpers.md (hunk 2 above)
Comment (borja-munoz) [reply docs/guides/02-using-style-helpers.md]: Added

**Pull request merged**